README.md

```diff
@@ -356,7 +356,7 @@ module "api_gateway" {
 ##### Inputs
 
 | Variable                                  | Type         | Description                                                                                                                                                                                                 | Required | Default                                                   |
-| ----------------------------------------- | ------------ |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------- | --------------------------------------------------------- |
+| ----------------------------------------- | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | --------------------------------------------------------- |
```

Comment:
just wondering, are the whitespace changes according to the rest of the file..?

Reply:
I think my IDE does that automatically for all markdown files, so it should be the same in the whole file 🤔